### PR TITLE
[26202] Improve pdf print layout of work packages fullscreen view

### PR DESCRIPTION
**Pull request by HDinger**
Improve PDF view for WP full screen.

https://community.openproject.com/projects/openproject/work_packages/26202/activity

- [ ] Remove box "Click to add comment"

### CODE REVIEW

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/02e9ba14558cb5e6553003fb33b11831e1579d90) (deployed commit: c2e4e8c59c1b558a191f4c91298a8313dc4f386a).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/7e627851131b10cca8162587bd8e706fa579949e) (deployed commit: 93f06bbdfea12507d12286d762d3568fd10b0496).

**Comment by lindenthal** [comment]
@HDinger looks great. Can you please also remove the following action links

- Add relation
- Add child (new/existing)
- Group by work package type/relations type

Thanks
Niels

**Comment by HDinger** [comment]
I included the feedback Niels mentioned. As discussed I am setting this on hold until we decided on how to go on with this.

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/54513af649117f4df04a26c58d048bb29487159e) (deployed commit: 89887ccd0a7c0b0f8cdfd5335bacba4bc8bdadc3).

**Comment by lindenthal** [comment]
@wielinde 

Can you please take care of this small change?

File: app/assets/stylesheets/content/work_packages/single_view/_single_view.sass [hunk 1]
```diff
@@ -209,7 +209,7 @@ i
     max-width: 55%
 
 // Implement two column layout for WP full screen view
-@media screen and (min-width: 92rem)
+@media screen and (min-width: 92rem), print
```
Comment (wielinde) [inline app/assets/stylesheets/content/work_packages/single_view/_single_view.sass:212]: @HDinger Nice!!

**Pull request merged**